app/javascript/app/components/emission-pathways-table/emission-pathways-table-component.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Table from 'components/table';
+import NoContent from 'components/no-content';
+import Search from 'components/search';
+// import Dropdown from 'components/dropdown';
+import darkSearch from 'styles/themes/search/search-dark.scss';
+import layout from 'styles/layout.scss';
+import EspModelsProvider from 'providers/esp-models-provider';
+import EspScenariosProvider from 'providers/esp-scenarios-provider';
+import EspIndicatorsProvider from 'providers/esp-indicators-provider';
+import styles from './emission-pathways-table-styles.scss';
+
+class EmissionPathwaysTable extends PureComponent {
+  getTableContent() {
+    const { loading, data, noContentMsg } = this.props;
+    if (loading) return null;
```

Comment:
Can you add a loader here please. Lots of examples elsewhere. Be sure to set a min height and look at absolute positioning when changing tabs.